netbox/dcim/api/serializers_/cables.py

```diff
@@ -16,14 +16,27 @@
     'CableSerializer',
     'CableTerminationSerializer',
     'CabledObjectSerializer',
+    'GenericObjectTerminationSerializer',
     'TracedCableSerializer',
 )
 
 
+class GenericObjectTerminationSerializer(GenericObjectSerializer):
```

Comment:
This doesn't require a new serializer: We can just add the `id` field to GenericObjectSerializer.

Reply:
Unfortunately since a_terminations/b_terminations is a property function that includes the actual termination (Interface/CircuitTermination/etc) and not the CableTermination it won't be the proper `id` (it will be the interface id for example, not the cable termination id).

This is why I opted to go this route, as as you mention, it would be a breaking change.  Perhaps we should aim for 4.1 instead and I just rebase and swap the serializer.

It may also make more sense to maintain this a_terminations/b_terminations and add a generic "terminations" which operates on `Cable.terminations` which is I think where it would be appropriate to use the CableTerminations serializer.